### PR TITLE
[SW-725] Select assets menu is not scrollable

### PR DESCRIPTION
**Pull request by ifaouibadi**

### CODE REVIEW

File: src/popup/components/ListItemWrapper.vue [hunk 1]
```diff
@@ -28,9 +28,7 @@ export default defineComponent({
 
 .list-item-wrapper {
   display: block;
-  padding: 8px var(--screen-padding-x);
-  margin-left: calc(-1 * var(--screen-padding-x));
-  margin-right: calc(-1 * var(--screen-padding-x));
```
Comment (CedrikNikita) [inline src/popup/components/ListItemWrapper.vue:33]: Why do you remove `calc`s here?

**Review by CedrikNikita — APPROVED** [review]
In general, code changes look good. In FF browser look fine for me, but i didn't test it heavily.

File: src/popup/components/ListItemWrapper.vue (hunk 1 above)
Comment (ifaouibadi) [reply src/popup/components/ListItemWrapper.vue:33]: this was adding -16px margins to left & right
and for padding the `var(--screen-padding-x)`is not working

Comment (CedrikNikita) [reply src/popup/components/ListItemWrapper.vue:33]: `var(--screen-padding-x)` is heavily used across our project, we shall adjust it everywhere.

**Pull request merged**